tests/testcases/export-ts/index.ts

```diff
@@ -0,0 +1,137 @@
+export type JSXElements = keyof JSX.IntrinsicElements;
```

Comment:
please give the testcase a more meaningful name, such as `preserve-tripleslash-references`